example-clients/ipload.c

```diff
@@ -199,6 +234,9 @@ main (int argc, char *argv[])
 		}
 	}
 
+	if (autoclose_opt)
+		jack_deactivate(client);
+
```

Comment:
Indentation is kind of hard to understand here, maybe add braces and fix indent of the next line (jack_client_close) (Yes this was broken before, but the additional condition makes it even harder to read in my eyes)